c/tools/msgr-send.c

```diff
@@ -289,7 +289,7 @@ int main(int argc, char** argv)
         check_messenger(messenger);
     }
 
-    LOG("Messages received=%llu sent=%llu\n", received, sent);
+    LOG("Messages received=%" PRIu64 " sent=%" PRIu64 "\n", received, sent);
```

Comment:
I don't think these C99 macros exist in MSVC before VS2013

Reply:
They don't, but it's in "platform/platform.fmt"

https://github.com/apache/qpid-proton/blob/97b62d2b7ce22c84e7d8db1ace8aa0355ea5fbe7/c/src/platform/platform_fmt.h#L61-L65

Reply:
True, but is that header included? It's internal to proton and (I think) this is code that only uses the exposed proton API (albeit the old messenger API.

Reply:
Ah - there are defines in c/tools/msgr-common.h which are included.

Reply:
Seems I was more lucky than knowledgeable, then.